### ISSUE TITLE
How to use nodegit-lfs with nodegit?

### ISSUE THREAD

**Issue by Githubber2021** [Axosoft/nodegit-lfs#30]
Hello,

I use [nodegit][1] for my NodeJS project and would like to add now Git LFS support. So I found [nodegit-lfs][2] and the description states:

> Nodegit LFS is an extension library used to augment NodeGit with the capability to use git lfs via the command line.
> To use this package, you must have git and git lfs installed.

Can anyone explain me the purpose behind this requirement? Nodegit itself works with libgit2, but to use Git LFS I need `git` and `git lfs` installed?

Does anyone know how for example GitKraken has solved this to fulfill their Git LFS support? Thanks a lot!

  [1]: https://github.com/nodegit/nodegit
  [2]: https://github.com/Axosoft/nodegit-lfs

**Comment by xmedeko** [comment]
Yes, GitKraken requires to have command line git and git-lfs installed in order to have LFS working, see https://help.gitkraken.com/gitkraken-client/git-lfs/#git-lfs-requirements

The reason is that git-lfs does not provide API through C library, see https://github.com/nodegit/nodegit/issues/914#issuecomment-301884119
